src/ddmd/backend/outbuf.d

```diff
@@ -26,16 +26,16 @@ struct Outbuffer
     ubyte *buf;         // the buffer itself
     ubyte *pend;        // pointer past the end of the buffer
     ubyte *p;           // current position in buffer
-    uint len;           // size of buffer
-    uint inc;           // default increment size
     ubyte *origbuf;     // external buffer
 
     //this();
 
-    this(size_t incx); // : buf(null), pend(null), p(null), len(0), inc(incx), origbuf(null) { }
+    this(size_t incx); // : buf(null), pend(null), p(null), origbuf(null) { }
```

Comment:
Please rename incx to initsz or initialSize.

Reply:
ok